__tests__/setup/waitForObservable.js

```diff
@@ -0,0 +1,27 @@
+import promiseRaceMap from 'promise-race-map';
+
+import observableToPromise from './observableToPromise';
+
+export default async function waitForObservable(observable, target) {
+  const observer = observableToPromise(observable);
+
+  try {
+    for (;;) {
```

Comment:
Just curious, is it better to use `for(;;)` than `while (true)`?

Reply:
I believe it is slim-to-zero difference.

`for (;;)` seems doesn't have a predicate when compare to `while (true)`. And after minify, `for(;;)` is a byte smaller than `while(1)`. 😉